### PR TITLE
Gulp & Eslint

### PR DESCRIPTION
**Pull request by JuhQ**
- Added gulp with eslint
- Fixed issues raised by eslint

### CODE REVIEW

**Comment by polygonplanet** [comment]
Thanks for PR.
* Please keep the trailing whitespace in .md file.

**Comment by polygonplanet** [comment]
chillout is not used task runner (gulp or grunt), I think that it can be completed in webpack's plugin.

File: src/nexttick.js [hunk 1]
```diff
@@ -25,14 +21,12 @@ const nextTick = (() => {
...
-      tail = tail.next = { task: task };
+      tail = tail.next = { task };
       channel.port2.postMessage(0);
     };
   }
 
-  return task => {
-    setTimeout(task, 0);
-  };
+  return task => setTimeout(task, 0);
```
Comment (polygonplanet) [inline src/nexttick.js]: These arrow functions are block statement because the nextTick is not needed return value for optimization.

**Comment by JuhQ** [comment]
Why the whitespace in .md? Imho unnecessary whitespace is scum of the earth

**Comment by polygonplanet** [comment]
The trailing 2 whitespace meens new line in .md

**Comment by JuhQ** [comment]
image

**Comment by polygonplanet** [comment]
This README.md is using the list style to reference APIs.
Should we improve the writing document reference format?
ss2016-10-25-170107-813

**Comment by JuhQ** [comment]
Hmm, documentation breaks if extra whitespace is removed?

**Comment by polygonplanet** [comment]
Yes.

ss2016-10-25-171039-815

**Comment by JuhQ** [comment]
Well damn! Would extra linebreak fix them? Seems odd that it's dependent on magically placed whitespace. Lot's of people (like myself) have their editors configured in a way which removes unnecessary whitespace on save, so adding that whitespace back will at some point be a problem again. I think it should be fixed now and not later :)

**Comment by polygonplanet** [comment]
Yes, I think too.
I think the trailing whitespace is unnecessary, and the document should be discribe simply without the trailing whitespace.
I'll fix later about document 😄

File: src/nexttick.js (hunk 1 above)
Comment (polygonplanet) [inline src/nexttick.js]: Thanks!

**Comment by polygonplanet** [comment]
Could you setting the ESLint using webpack' eslint-loader without gulp?

**Comment by polygonplanet** [comment]
OK, I'll use gulp for linting.
Can you revert the .md files (README.md and README-ja.md) to no changes?
I will resolve the trailing whitespace probrem at later.

**Comment by polygonplanet** [comment]
Confirmed, thank you!

**Pull request merged**